mailpoet/lib/AdminPages/Pages/Help.php

```diff
@@ -128,10 +128,10 @@ private function getLatestActionSchedulerActionDate(string $hook, string $status
     return null;
   }
 
-  private function buildTaskData(ScheduledTaskEntity $task): array {
+  public function buildTaskData(ScheduledTaskEntity $task): array {
```

Comment:
Another small detail, but it seems to me that the title of this commit is a bit misleading. It mentions moving buildTaskData() to Pages\Help, but the method already exists in this class.

Reply:
Thanks for noticing this @rodrigoprimo, adjusted the commit title.